C7Engine/C7Settings.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using IniParser.Exceptions;
+
+namespace C7Engine
+{
+	using IniParser;
+	using IniParser.Model;
+	
+	public class C7Settings {
+		private const string SETTINGS_FILE_NAME = "C7.ini";
+		public static IniData settings;
+
+		public static void LoadSettings() {
+			try {
+				settings = new FileIniDataParser().ReadFile(SETTINGS_FILE_NAME);
+			}
+			catch(ParsingException ex) {
+				//First run.  The file doesn't exist.  That's okay.  We'll use sensible defaults.
+				settings = new IniData();
+				SaveSettings();
+			}
+		}
+
+		public static void SaveSettings() {
+			new FileIniDataParser().WriteFile(SETTINGS_FILE_NAME, settings);
+		}
+
+		public static void SetValue(string section, string key, string value)
+		{
+			settings[section][key] = value;
```

Comment:
I haven't looked at the API- does this create the section/key if they don't already exist?

Reply:
It does.  After running this branch on a fresh computer, I see the following in the AI file:

```
[audio]
musicVolume = 100
```

I haven't dug into how they got it to work like that, but I like what I've seen so far.